pkg/osbuild/oscap_autotailor_stage.go

```diff
@@ -27,6 +33,17 @@ func (c OscapAutotailorConfig) validate() error {
 	if c.NewProfile == "" {
 		return fmt.Errorf("'new_profile' must be specified")
 	}
+	for _, override := range c.Overrides {
+		if _, ok := override.Value.(uint64); ok {
+			continue
+		}
+
+		if _, ok := override.Value.(string); ok {
+			continue
+		}
+
+		return fmt.Errorf("override 'value' must be an integere or a string")
```

Comment:
would it make sense to also include the name of the override value in the error?